### PR TITLE
chore: Fix support for Node.js 18 with yarn

### PR DESCRIPTION
**Pull request by slaweet**
`yarn install` was failing with the following error:

```
error @achrinza/event-pubsub@5.0.6: The engine "node" is incompatible with this module. Expected version "13 || 14 || 16 || 17". Got "18.7.0"
error Found incompatible module.
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "node": "14 || 16 || 17 || 18"
   },
   "dependencies": {
-    "@achrinza/event-pubsub": "5.0.6",
+    "@achrinza/event-pubsub": "https://github.com/achrinza/event-pubsub#4d0a3e3",
```
Comment (slaweet) [inline package.json:22]: TODO change here and re-generate `package-lock.json` when the change from https://github.com/achrinza/event-pubsub/pull/9 is released

```suggestion
    "@achrinza/event-pubsub": "5.0.8",
```

**Comment by achrinza** [comment]
Thanks for opening a PR! I think it'll be better to release `@achrinza/event-pubsub` then bump the version from there since it s also under my purview.


Currently I can't cut any releases this week as I'm away for conscription, but i'll work towards publishing the patches this weekend.

In the mean time, you can use Yarn's [selective dependency resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/) to mimic this PR.

**Comment by Form1ca** [comment]
any progress?

**Comment by achrinza** [comment]
Hi @Form1ca, apologies for the delayed reply.

I believe Node.js v18 and v19 support has already been propagated to all the necessary packages:

https://github.com/achrinza/node-ipc/issues/40#issuecomment-1288042345

**Comment by achrinza** [comment]
Closing this as per-above.

**Pull request closed without merging**